mobile-ui/src/main/java/com/playone/mobile/ui/firebase/v1/PlayoneFirebaseV1.kt

```diff
@@ -188,5 +444,23 @@ class PlayoneFirebaseV1(
         onDataChange = { strategy?.apply { callback(this(it)) } }
         onCancelled = { it.makeCallback(errorCallback) }
     }
+
+    private fun <D> DatabaseReference.runTransaction(
```

Comment:
if you think the extensions can be moved to other places this one should do as well.

Reply:
Those extension functions are just for this file temporally. I have no idea whether they will be used by some file or not now.
That's why I didn't separate them to the other file.

Reply:
FYI, https://kotlinlang.org/docs/reference/extensions.html#scope-of-extensions